### PR TITLE
New package: xforms-1.2.4

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

**Comment by Vaelatern** [comment]
https://github.com/voidlinux/void-packages/pull/4241#issuecomment-222337728

File: srcpkgs/xforms/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'xforms'
+pkgname=xforms
+version=1.2.4
+revision=1
+build_style=gnu-configure
+homepage="http://xforms-toolkit.org/"
+distfiles="http://download.savannah.gnu.org/releases/xforms/xforms-${version}.tar.gz"
+makedepends="libjpeg-turbo-devel libX11-devel libXpm-devel"
+depends="libjpeg-turbo libX11 libXpm"
```
Comment (Vaelatern) [inline srcpkgs/xforms/template]: Remove this line

**Comment by Vaelatern** [comment]
The rule within Void is that for every template altered, there should be one commit. Those commits should be named as directed by the end of the [Manual (link)](https://github.com/voidlinux/void-packages/blob/master/Manual.md#contributing-via-git). You will probably need to rebase your branch on top of master (by `git checkout xforms124; git pull --rebase origin`) to clear the merge commit. You can `git rebase --interactive HEAD~5` before that action to be able to delete, re-order, or squash commits.

Only one PR, please, unless you are told otherwise. Unrelated features, such as if you fixed something in `firefox` and added a new spotify client, should go in separate PRs. If you wanted to package something, and it depended on a different package not yet in Void, you would put both of those package additions in the **same** PR.

**Comment by abenson** [comment]
Sorry, it took me a few to figure out how to fix this. Had to `git rebase -i <sha>` and then force a push. I'm still learning some of the less straightforward git actions.

**Comment by Vaelatern** [comment]
You can also `git rebase -i origin/master` if voidlinux/void-packages is origin, just fyi.

File: srcpkgs/xforms/template [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'xforms'
+pkgname=xforms
+version=1.2.4
+revision=1
+build_style=gnu-configure
+homepage="http://xforms-toolkit.org/"
+distfiles="http://download.savannah.gnu.org/releases/xforms/xforms-${version}.tar.gz"
+makedepends="libjpeg-turbo-devel libX11-devel libXpm-devel"
```
Comment (Vaelatern) [inline srcpkgs/xforms/template]: `libjpeg-turbo-devel libXpm-devel` since they eventually pull in `libX11-devel`

**Comment by Gottox** [comment]
@Vaelatern any further comments on this PR? looks okay to me.

File: srcpkgs/xforms/template [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+pkgname=xforms
+version=1.2.4
+revision=1
+build_style=gnu-configure
+homepage="http://xforms-toolkit.org/"
+distfiles="http://download.savannah.gnu.org/releases/xforms/xforms-${version}.tar.gz"
+makedepends="libjpeg-turbo-devel libXpm-devel"
+short_desc="Graphical uesr interface toolkit for X11"
+maintainer="Andrew Benson <andrew@rockstarunix.org>"
+license="GPL-2"
```
Comment (Vaelatern) [inline srcpkgs/xforms/template]: `LGPL-2`

**Pull request closed without merging**